src/cli/README.md

```diff
@@ -0,0 +1,60 @@
+# LIBPLCTAG Command-Line Interface
+
+This is a command line program extension to the _libplctag_. There are three basic types of operation supported by the program:
+
+- __READ__: read the value of the tag specified once and return the value read.
+- __WRITE__: write the value provided to the tag specified once and return the resulting value of the tag.
+- __WATCH__: read the value of the specified tag on start and return it. Then perform subsequent periodic reads for the interval specified and only return the value if it has changed since the last known value.
+
+## Pre-requisites
+The CLI is pre-compiled with all the library source code, so there are typically no required dependencies on Linux/MacOS.
+
+On Windows make sure that the Visual C Runtime is present on the system path. The specific file required is _vcruntime140.dll_.
+
+## Usage
+To use this CLI you need to specify and run a certain operation on each execution, with the specific _libplctag_ parameters for your specific PLC. Below is the description for the available command-line options. 
+
+```
+cli {--read | --write | --watch} {-protocol} {-ip} {-path} {-plc}
+                [-debug] [-interval] [-attributes] [-offline]
+
+        CLI Action (Required):
+
+        --read          - Perform a one-shot READ operation.
+        --write         - Perform a one-shot WRITE operation.
+        --watch         - Perform a continuous WATCH operation at the specified interval.
+        -h | --help     - Prints the Usage details.
+
+        LIBPLCTAG Parameters (Required):
+
+        -protocol       - type of plc protocol. (default: ab_eip)
+        -ip             - network address for the host PLC. (default: 127.0.0.1)
+        -path           - routing path for the Tags. (default: 1,0)
```

Comment:
Some PLC types explicitly warn if they have a path.  For instance all the PCCC-based PLCs.